src/inverseRobot.js

```diff
@@ -24,7 +24,27 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const reverseR = {};
+  let counter = 0;
+
+  for (const key in robot) {
+    for (const item in robot) {
+      if (robot[key] === robot[item]) {
+        counter++;
+      }
+    }
+
+    if (counter > 1) {
+      return null;
+    }
+    counter = 0;
+  }
```

Comment:
This cycle in redundant in your code
You can if statement for the next cycle and check if the property is already exists in robot - return null